src/components/floweditor/FlowEditor.helper.ts

```diff
@@ -106,7 +108,7 @@ export const setConfig = (uuid: any) => {
 };
 
 export const loadfiles = (startFlowEditor: any) => {
-  (window as any).static_url = '/sitestatic/';
+  (window as any).static_url = '/static/';
```

Comment:
Can you check if we need this? If not then just remove.

Reply:
we dont, removed this